src/main/java/com/zufar/onlinestore/cart/endpoint/CartEndpoint.java

```diff
@@ -32,6 +36,16 @@ public class CartEndpoint {
 
     private final CartApi cartApi;
 
+    @PostMapping(value = "/api/v1/cart/items/")
```

Comment:
pls use just /items here

Reply:
It is related to cart. I don't agree with you